docs/getting_started.rst

```diff
@@ -0,0 +1,79 @@
+
+.. py:currentmodule:: rticonnextdds_connector
+
+Getting Started
+===============
+
+Installing RTI Connector for Python
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Make sure you have Python installed. Then use *pip* to install *RTI Connector*
+as follows:
```

Comment:
There are then 2 ways to obtain RTI Connector, installing it via pip:
....
or cloning the github repository:
 ....
In order to gain access to the example explained in this Getting Started Guide, clone the github repositiory.